LICENSE.md

```diff
@@ -0,0 +1,19 @@
+As a work of the United States Government, this project is in the public domain within the United States.
+
+## Restrictions on FEC data
```

Comment:
I think this was a copy/paste issue, the paragraph should be dropped. I'll fix it...